app/_landing_pages/dev-portal/self-service.yaml

```diff
@@ -0,0 +1,213 @@
+                  When RBAC is enabled, any Dev Portal teams and roles you apply to a developer will control their access.
+  - header:
+      type: h3
+      text: "Get started with user authentication"
+    columns:
+      - blocks:
+          - type: card
+            config:
+              title: Configure Dev Portal SSO
+              description: |
+                Set up SSO for the {{site.konnect_short_name}} Dev Portal using OpenID Connect (OIDC) or SAML.
+              icon: /assets/icons/lock.svg
+              cta:
+                url: /dev-portal/sso/
+      - blocks: 
+          - type: card
+            config:
+              title: Dev Portal IdP team mappings
+              description: |
+                Map existing developer teams from a third-party identity provider (IdP) and their permissions to elements in a {{site.konnect_short_name}} Dev Portal.
+              icon: /assets/icons/team.svg
+              cta:
+                url: /dev-portal/team-mapping/
+      - blocks:
+          - type: card
+            config:
+              title: Dev Portal RBAC
+              description: |
+                Learn about Dev Portal pre-defined teams and roles for RBAC.
+              icon: /assets/icons/rbac.svg
+              cta:
+                url: /konnect-platform/teams-and-roles/#dev-portal
+  
+  - header:
+      type: h2
+      text: "Application authentication strategies"
+    columns:
+      - blocks:
+        - type: structured_text
+          config:
+            blocks:
+              - type: text
+                text: |
+                  Application authentication allows developers to authenticate with your API using credentials. 
+                  Developers use the credentials from the authentication strategy when they use an API from your Dev Portal.
+                  You can define and reuse multiple authentication strategies for different APIs and Dev Portals.
+                  
+                  When you select an [authentication strategy](/dev-portal/auth-strategies/) during [API publication](/dev-portal/apis/) to a Dev Portal, {{site.konnect_short_name}} automatically applies the strategy to the linked Gateway Service.
+
+                  Dev Portal supports the following authentication strategies:
+                  * [Key authentication (`key-auth`)](/dev-portal/auth-strategies/#configure-the-key-auth-strategy)
+                  * [OpenID Connect (`oidc`)](/dev-portal/auth-strategies/#dev-portal-oidc-authentication)
+                  * [Dynamic Client Registration (DCR)](/dev-portal/dynamic-client-registration/)
+
+                  If a Gateway Service isn't associated with the API when you choose an authentication strategy, the settings are saved and applied once a Service is linked. 
+                  If a Service is later unlinked, the authentication strategy is applied to the next linked Service.
+  - header:
+      type: h3
+      text: "Get started with dynamic client registration"
```

Comment:
Should DCR be capitalized all the time, or lowercase? There's a mix here, though looks like it's mostly capitalized.